lib/bus.js

```diff
@@ -1,11 +1,11 @@
-const isPlainObject = require('lodash/isPlainObject');
+const isPlainObject = require('is-plain-obj');
```

Comment:
Why? `lodash` was fine. Since it is used by a lot of modules, the dedup avoid extra copies

Reply:
dang. I had some discussion with people about install size and went this route to appeal to the position that the smaller install size was better in this case. I'll have to revisit that